recipes/wildmidi/all/conanfile.py

```diff
@@ -0,0 +1,94 @@
+from conan import ConanFile
+from conan.tools.microsoft import is_msvc
+from conan.tools.files import export_conandata_patches, get, copy, rmdir
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+import os
+
+
+required_conan_version = ">=1.53.0"
+
+
+class WildmidiConan(ConanFile):
+    name = "wildmidi"
+    description = "WildMIDI is a simple software midi player which has a core softsynth library that can be used in other applications."
+    license = "LGPL-3.0-only"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.mindwerks.net/projects/wildmidi"
+    topics = ("audio", "midi", "multimedia", "music", "softsynth", "sound", "synth")
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+
+    def export_sources(self):
+        export_conandata_patches(self)
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+        self.settings.rm_safe("compiler.libcxx")
+        self.settings.rm_safe("compiler.cppstd")
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
+
+    def generate(self):
+        tc = CMakeToolchain(self)
+        if self.settings.os == "Windows":
```

Comment:
@jcar87 the behavior of CMakeToolchain is a little wonky here if you can take a look please

Reply:
I think this is more a problem with the project than CMakeToolchain. The project seems to incorrectly handle multi-configuration generators, like the Visual Studio generator.

Reply:
Absolutely agreed but I am curious if this is the right solution of perhaps patches might be more robust... maybe fixing upstream could be an option?

Reply:
So the right thing to do is to add a proper support for multi-config. I will try to make a change in upstream and come back with new PR when new version comes out.

Reply:
I definitely think upstream could be better but I also think this is an edge case the new Conan generators weren't tested with so it would be nice to at least have test code to make sure this is intended 

So I want to be clear, this PR is good :)